helpers/getReadingTime.js

```diff
@@ -1,5 +1,16 @@
-const getReadingTime = text => {
+export const getDescription = (body) => {
+  let articleText = '';
+  body.map((paragraph) => {
+    articleText += paragraph.text;
+    return articleText;
+  });
+  return articleText;
+};
+
+
+const getReadingTime = body => {
   try {
+    const text = getDescription(JSON.parse(body).blocks);
```

Comment:
Why are you converting the body into a js object?

Reply:
This is the main reason to come back to the backend. We are no longer receiving a text from the body of the article but an object which needs to be stringified to be saved in our database. To get our object again we are parsing it into an object

Reply:
About test, I am still fixing them cause many of them depend on the previous version of our body